### PR TITLE
lightdm: version bump to 1.19.5

### PR DESCRIPTION
**Pull request by rtlanceroad**

### CODE REVIEW

**Comment by rtlanceroad** [comment]
Retest please

**Comment by rtlanceroad** [comment]
Please retest

**Comment by rtlanceroad** [comment]
Retest

**Comment by rtlanceroad** [comment]
Please retest.

**Comment by lunar-linux-b01** [comment]
SUCCESS!

**Comment by rtlanceroad** [comment]
If someone merge this, I can retest lightdm-gtk-greeter PR.

File: x11-utils/lightdm/BUILD [hunk 1]
```diff
@@ -1,10 +1,4 @@
 OPTS+=" --disable-static \
         --disable-tests" &&
 
-default_build &&
-
-install -m 755 $SCRIPT_DIRECTORY/Xsession /etc/lightdm/Xsession &&
-
-# PolicyKit
-install -dm 750 -g 102 /usr/share/polkit-1/rules.d &&
-install -m 644 $SCRIPT_DIRECTORY/lightdm.rules /usr/share/polkit-1/rules.d/lightdm.rules
```
Comment (Ratler) [inline x11-utils/lightdm/BUILD]: @rtlanceroad what was the reason to remove the polkit rules? It was used to be control some functions like controlling upower etc (reboot, shutdown, etc). Does it have other means to deal with that now?

**Comment by Ratler** [comment]
I'm going to squash merge your changes and add a few more changes on top. pam needs a few tweaks and the Xsession wrapper script is needed, the default config also require a few changes.

**Comment by rtlanceroad** [comment]
I feel so. This module needs a lot of rework to get it functional. I did this yesterday just because Xsession and polkit rule don't exist in $SCRIPT_DIRECTORY, which caused build failure.

**Comment by Ratler** [comment]
Yes, after looking at the module I noticed that it couldn't have worked for quite a while. Hopefully I'll be done with the changes I have in mind later today.

**Comment by Ratler** [comment]
Rebased and merged with commits https://github.com/lunar-linux/moonbase-other/commit/a899da9a7a75e00a46ece589172af693f58ef4d8 and https://github.com/lunar-linux/moonbase-other/commit/ebbb37c20abb1eb258af826d1236ab254accc9ed

**Pull request closed without merging**